### PR TITLE
Update bootstrap.html - Changing navbar toggle icon to white

### PR DESCRIPTION
**Pull request by mayonnaisecolouredbenz7**
Changing the navbar toggle icon to white as is the case in other templates.  

I attempted to create the variable in the css file that the template is referring to originally _bs-navbar-toggler-icon-bg_.  However it wasn't overriding the variable shown in the console:
image

So it was easier to explicitly change the background-image property in the navbar-toggler-icon class in html file

### CODE REVIEW

**Comment by philippjfr** [comment]
pre-commit.ci autofix

**Comment by philippjfr** [comment]
>I attempted to create the variable in the css file that the template is referring to originally bs-navbar-toggler-icon-bg.

I suspect the problem is that you have to run a command to bundle the CSS files when you want them to be reflected using `panel bundle --templates --only-local` (which is not documented). Overall I'd prefer using the variable.

**Comment by mayonnaisecolouredbenz7** [comment]
Made the necessary changes

File: panel/template/bootstrap/bootstrap.html [hunk 1]
```diff
@@ -60,10 +60,11 @@
       }
       #header {
 	--bs-body-bg: var(--header-background);
+
```
Comment (philippjfr) [inline panel/template/bootstrap/bootstrap.html:63]:
```suggestion
```

**Review by philippjfr — APPROVED** [review]

**Comment by philippjfr** [comment]
Thank you @mayonnaisecolouredbenz7!

**Pull request merged**